Gems/ROS2/Code/Source/Camera/CameraSensor.h

```diff
@@ -16,31 +17,32 @@
 
 namespace ROS2
 {
+    struct CameraConfiguration
+    {
+        float verticalFieldOfViewDeg = 90.0f;  //!< vertical field of view of camera sensor.
+        int width = 640; //!< camera image width in pixels.
+        int height = 480; //!< camera image height in pixels.
+        Camera::ImagePostProcessingFn imagePostProcessingFunction; //!< post-processing function to apply to data.
```

Comment:
uber nit (repeated): Prefer sentence casing (starting with capital letter) for API comments.